packages/admin/cms-admin/src/redirects/RedirectActiveness.tsx

```diff
@@ -57,6 +58,7 @@ const RedirectActiveness = ({ redirect }: RedirectActivenessProps): JSX.Element
                     __typename: "Redirect",
                     id: redirect.id,
                     active: active,
+                    activatedAt: new Date(),
```

Comment:
```suggestion
                    activatedAt: active ? new Date() : null,
```

This should consider the active state.